src/FoundryCSS/ui/components/ReadMore/_read-more-config.scss

```diff
@@ -0,0 +1,8 @@
+/*
+ * Read More Config
+ *
+ * 1. Max Height
+ *
+ */
+
+$readmore-max-height: 12rem !default;
```

Comment:
12rem, or should this be a factor of a base spacing value somewhere up the chain?

Reply:
As a configurable item, this is probably fine to be a dumb value? I suspect the max-height would be different on most projects and need some specific value anyway?

Reply:
I think this one is fine as it's one of the very few places we ever specify a fixed height on something?